### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by Sasha380**

### CODE REVIEW

**Comment by SvitlanaLN** [comment]
Try to write messages of the program only in English. Also the name of namespace is in English too.

**Review by SvitlanaLN — COMMENTED** [review]
Try to write messages of the program only in English. Also the name of namespace is in English too.

File: Projects/Team1/Sasha Yakimchuk/Задача 202.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
+﻿using System;
+
+namespace Задача
+{
+    class MyProject
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Сколько денег вам надо перевести?");
```
Comment (romanukhaav) [inline Projects/Team1/Sasha Yakimchuk/Задача 202.cs:9]: https://translate.google.com.ua/ - try it please ;)

**Review by romanukhaav — CHANGES_REQUESTED** [review]
please in English

**Pull request closed without merging**